packages/core/src/components/Drawer/Drawer.tsx

```diff
@@ -94,39 +84,14 @@ export const HvDrawer = (props: HvDrawerProps) => {
     ...others
   } = useDefaultProps("HvDrawer", props);
 
-  const { classes, css, cx } = useClasses(classesProp);
-  const { colors, activeTheme, rootId } = useTheme();
-
-  const [backgroundColorValue, setBackgroundColorValue] = useState<string>(
-    getVarValue(theme.drawer.backDropBackgroundColor, rootId) || ""
-  );
+  const { classes, cx } = useClasses(classesProp);
 
   const closeButtonDisplay = () => <Close role="none" />;
 
   const CloseButtonTooltipWrapper = buttonTitle
     ? withTooltip(closeButtonDisplay, buttonTitle, "top")
     : closeButtonDisplay;
 
-  const [backgroundColor, setBackgroundColor] = useState(
-    getBackgroundColor(backgroundColorValue)
-  );
```

Comment:
We don't need this anymore?

Reply:
The drawer backdrop color isn't working probably for a while, so it wasn't making a difference, I tried for a few minutes to fix it but decided it's probably best to open a separate issue

Reply:
agreed 👍